src/olympia/core/apps.py

```diff
@@ -98,6 +99,26 @@ def static_check(app_configs, **kwargs):
     return errors
 
 
+@register(CustomTags.custom_setup)
+def db_charset_check(app_configs, **kwargs):
```

Comment:
Because we are now using `reset_db` we rely on django-extensions to create the Database. This ensures that when that happens, it has the correct charset.